src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private String[] colors = {"red", "blue", "green", "yellow", "pink", "white", "black"};
+
+    public String getRandomColor() {
+        Random random = new Random();
```

Comment:
The `Random` instance should be a class-level variable instead of being created each time the `getRandomColor` method is called. This is to avoid unnecessary creation of `Random` instances which can be resource-intensive.